LayoutTests/fast/events/pointer/ios/drag-populates-pointer-events-with-movementxy-fields.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <script src="../../../../resources/js-test-pre.js"></script>
```

Comment:
This can just be `<script src="../../../../resources/js-test.js"></script>`, and you can remove the `js-test-post.js` below.

Reply:
Thanks, fixed in https://github.com/WebKit/WebKit/commit/eed593fcf230c5752fc982cef0b7cbdda782897a. When do we need to use the `pre`/`post` scripts instead?

Reply:
I'm not sure there is ever a need for them over `js-test.js`. My understanding is that they are just the old way of doing things.